File: Python/Gate four/phase_gate4.py
def is_prime(number):
    if number <= 1:
        return False
    for count in range(2, number):
        if number % count == 0:
            return False
    return True

def collect_primes(numbers):
    primes = []
    for num in numbers:
        if is_prime(num):
            primes.append(num)
    return primes

File: Java/Gate Four/PhaseGate4.java
import java.util.Arrays;
public class PhaseGate4 {

    public boolean isPrime(int number){
        if(number <= 1) return false;
           
        for(int count = 2; count < number; count++){
            if(number % count == 0) return false;
            }            
            return true;    
        }
        
    public int[]getSortedPrimes(int[]numbers){

        int[] temp = new int[numbers.length];
        int index = 0;        

        for(int num : numbers){
            if(isPrime(num)){
                temp[index++] = num;
                
                }
            }
            
    int[] primes = Arrays.copyOf(temp, index);
    Arrays.sort(primes);
    return primes;
    } 
}
